.github/workflows/ci.yml

```diff
@@ -8,24 +8,16 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version:
-        - 12
-        npm-test:
-        - i18n_extract
-        - is-es5
-        - lint
-        - test
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
+    - run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
+    - name: setup node
+      uses: dcodeIO/setup-node-nvm@master
       with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install -g npm@6
+       node-version: "${{ env.NODE_VER }}"
     - run: make requirements
     - run: make test NPM_TESTS=build
-    - run: make test NPM_TESTS=${{ matrix.npm-test }}
+    - run: make test NPM_TESTS="i18n_extract is-es5 lint test"
```

Comment:
Since we've dropped support for IE 11, it should safe to remove the `is-es5` check (and the associated NPM script and `es-check` package) since our other supported browsers are [natively compatible with ES6](https://www.w3schools.com/js/js_versions.asp).

Reply:
ah that is indeed a good point! I just was not sure if I wanted to do it for this repo for this PR, but if @davidjoy is okay with that I can include it in there

Reply:
Yeah, we can do that. 👍🏻  Will be a nice cleanup.

Reply:
done!